modules/understanding-agent-install.adoc

```diff
@@ -65,6 +65,7 @@ In the `install-config.yaml`, specify the platform on which to perform the insta
 
 * `baremetal`
 * `vsphere`
+* `external`
```

Comment:
Do I need to add a note or any clarification that this is for OCI only?

Reply:
No, agent based installer does support OCI and generic  external platforms. https://issues.redhat.com/browse/AGENT-704 and https://issues.redhat.com/browse/AGENT-729